reference/constraints/Email.rst

```diff
@@ -91,14 +91,34 @@ Basic Usage
 Options
 -------
 
+mode
+~~~~
+
+**type**: ``string`` **default**: ``loose``
+
+This option is optional and pattern the email address is validated against.
+Valid values are:
+
+* ``loose``
+* ``strict``
+* ``html5``
+
+loose
+.....
+
+A simple regular expression. Allows all values with an "@" symbol in, and a "."
+in the second host part of the email address.
```

Comment:
e-mail

Reply:
I had a look across the other documentation as to what we tend to use, and it seems to be "email".

I've switched everything to email in this document, plus the other email related document, so we're consistent with everything else.